transactor/src/main/scala/io/mediachain/transactor/Dummies.scala

```diff
@@ -17,19 +19,20 @@ object Dummies {
     val CBORType = "" // unused
     override def toCbor = CMap.withStringKeys("@link" -> CString(this.toString))
   }
-  
+
   class DummyStore extends Datastore {
     var seqno = 0
-    val store: MMap[Reference, DataObject] = new MHashMap
-    
-    override def put(obj: DataObject): Reference = {
-      val ref = new DummyReference(seqno)
+    val store: MMap[MultiHash, DataObject] = new MHashMap
```

Comment:
and here, why this? We still need Reference -> Datastore mapping in order to be able to extract values from the datastore for testing.